docs/configuration.md

```diff
@@ -25,6 +25,16 @@ The majority of the configuration is around
   connected with the house and its activities. They are sent to guests in the
   welcome email so make sure to populate these. 
 
+## Stripe in development
+
+If you want to test Stripe payments, you need to set up an account and get some test keys.
+
+First, [create an account](https://dashboard.stripe.com/register). On the left-hand menu, click "Developers", then "API Keys".
+
+**Important:** Make sure the keys start with `pk_test_` and `sk_test_` otherwise you will be charging real credit cards!
```

Comment:
@bfirsh Just checking here, are these test keys sensitive in anyway?

Reply:
Yes, somewhat. They can't be used to charge cards, and the data seems to be separated from the live data, but they are still associated with a real Stripe account.

It seems prudent to get everyone to set up their own Stripe accounts and generate their own keys, at the very least so development environments don't collide with each other.